app/src/androidTest/java/com/github/fribourgsdp/radio/activities/WorkingJoinGameActivityTest.kt

```diff
@@ -32,15 +35,16 @@ import org.junit.runner.RunWith
  */
 @RunWith(AndroidJUnit4::class)
 class WorkingJoinGameActivityTest {
-
-    @get:Rule
-    var mRuntimePermissionRule = GrantPermissionRule.grant(Manifest.permission.CAMERA)
     @get:Rule
-    var mRuntimePermissionRule2 = GrantPermissionRule.grant(Manifest.permission.VIBRATE)
```

Comment:
Those lines were the ones that were making the tests in pr #216 timeout. As discussed with @Flamantrose321, they can be removed since those permissions were mocked. I don't know why it wasn't an issue before and why it only occurred in my case but at least now there shouldn't be any problem anymore with this.